src/Formatter/DateTimeFormatter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Formatter;
+
+use Symfony\Component\PropertyAccess\PropertyPath;
+
+final class DateTimeFormatter implements DataFormatterInterface
+{
+    /** @var null|string */
+    private $dateTimeFormat = null;
```

Comment:
ah we use it here already, too

https://github.com/sonata-project/exporter/pull/147/files#diff-59abbb9207041c402aca780bf90bc52fR47

so please ignore my comment